src/community/newsletter.njk

```diff
@@ -32,15 +32,16 @@ hubspot:
                 </div>
                 <div class="flex flex-col md:flex-row">
                     <div class="pr-2 md:w-1/3">
-                        {% if item.data.image %}
-                            <div class="w-full h-auto mb-10">
+                        <div class="w-full h-auto mb-10">
```

Comment:
Can we make this a partial or shortcode? Not sure what's more appropriate on a tehcnical level, but feels like if we DRY our code, it's easier going forward

Ref: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself